src/Cache/Cache.php

```diff
@@ -223,7 +223,7 @@ public static function engine(string $config): CacheEngine
      */
     public static function pool(string $config)
     {
-        return new SimpleCacheEngine(static::engine($config));
+        return static::engine($config);
```

Comment:
`static::engine()` deprecated

Reply:
Yes. That is also on my list of things to change, but I wanted to take small steps.

Reply:
so , goods for this step :+1: